--- firestarter-0.9.2/configure.in~	Fri Jul 23 20:34:33 2004
+++ firestarter-0.9.2/configure.in	Fri Jul 23 20:34:33 2004
@@ -36,7 +36,7 @@
 AM_GCONF_SOURCE_2
 
 dnl i18n
-ALL_LINGUAS="de es fr no ru sv fi da it ja ga uk hu pt_BR nl pl ca sk sr ro ko el az tr pt zh_TW vi zh_CN"
+ALL_LINGUAS="de es fr nb ru sv fi da it ja ga uk hu pt_BR nl pl ca sk sr ro ko el az tr pt zh_TW vi zh_CN"
 GETTEXT_PACKAGE=firestarter
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE",[Useful for glib-gettextize])
