diff -urN firestarter-0.9.3.orig/firestarter.desktop firestarter-0.9.3/firestarter.desktop
--- firestarter-0.9.3.orig/firestarter.desktop	2004-10-05 17:31:07.292019248 +0000
+++ firestarter-0.9.3/firestarter.desktop	2004-10-05 17:32:02.357648000 +0000
@@ -58,6 +58,7 @@
 Encoding=UTF-8
 StartupNotify=true
 Terminal=false
-Categories=System;Application;
+Categories=GTK;GNOME;Utility;SystemSetup;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=firestarter
+# vi: encoding=utf-8
