diff -Nur firestarter-1.0.3.orig/firestarter.desktop.in firestarter-1.0.3/firestarter.desktop.in
--- firestarter-1.0.3.orig/firestarter.desktop.in	2005-01-29 13:32:09.000000000 +0100
+++ firestarter-1.0.3/firestarter.desktop.in	2005-04-10 02:46:55.000000000 +0200
@@ -7,6 +7,7 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=System;Application;
+Categories=GTK;GNOME;Utility;System;Security;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=firestarter
+# vi: encoding=utf-8
