--- firestarter-1.0.3/src/netfilter-script.c.orig	2005-01-29 13:32:08.000000000 +0100
+++ firestarter-1.0.3/src/netfilter-script.c	2005-05-06 03:50:46.000000000 +0200
@@ -405,7 +405,7 @@
 			 "	$MPB ip_nat_irc 2> /dev/null\n"
 			 "fi\n");
 
-	fprintf (script, "if [ \"EXT_PPP\" = \"on\" ]; then\n"
+	fprintf (script, "if [ \"$EXT_PPP\" = \"on\" ]; then\n"
 			 "	$MPB bsd_comp 2> /dev/null\n"
 			 "	$MPB ppp_deflate 2> /dev/null\n"
 			 "fi\n\n");
